test/web-platform-tests/to-run.yaml

```diff
@@ -1190,8 +1189,8 @@ template-element/template-content-hierarcy.html: [fail, "We do not implement htt
 DIR: html/semantics/selectors
 
 pseudo-classes/checked-type-change.html: [fail, CSS computed style computation]
-pseudo-classes/dir-html-input-dynamic-text.html: [fail, need nwsapi fix]
-pseudo-classes/dir.html: [fail, https://github.com/dperini/nwsapi/issues/45]
+pseudo-classes/dir-html-input-dynamic-text.html: [fail, Unknown]
+pseudo-classes/dir.html: [fail, Unknown]
```

Comment:
Tests passes in internal tests of dom-selector, but fails when running WPT.
I don't know why...